### PR TITLE
Rn 2024

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/release-2024-0/platform-2024-0.md [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+<td>Administration</td>
+<td>Fixed an issue with the names of the files downloaded from the platform (for example, from the file repository or from event attachments). UTF-8 characters, for example, in the Japanese localization are no longer missing if the file name includes special characters like "+".</td>
+<td>MTM-53056</td>
+<td>10.18.90.0</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Administration</td>
+<td>Updating a custom alarm property, no longer creates a redundant audit log entry with misleading information about alarm clearance.</td>
```
Comment (rssag) [inline content/release-2024-0/platform-2024-0.md:230]:
```suggestion
<td>Updating a custom alarm property no longer creates a redundant audit log entry with misleading information about alarm clearance.</td>

```

File: content/release-2024-0/applications-2024-0.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+<td>Java SDK</td>
+<td>The dependency on <code>org.json:json</code> has been removed from the Java SDK.</td>
+<td>MTM-53340</td>
+<td>10.18.83.0</td>
+<td>Core</td>
+</tr>
+
+<tr>
+<td>Microservice SDK</td>
+<td>Dependencies on <code>org.eclipse.jetty:jetty-*</code> were updated to version 9.4.51.v20230217.</td>
```
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/release-2024-0/applications-2024-0.md:160]: You are right. I would leave this as this seem to be the version name.

File: content/release-2024-0/devices-2024-0.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+<td>The performance of the <code>migrateLwm2mDevices</code> operation has been improved. New command line arguments have been introduced with the operation. A list of legacy LWM2M devices can be specified directly from the shell command. Moreover, the migration of the LWM2M client registration objects can be skipped b< using an argument. For details, refer to <a href="https://cumulocity.com/guides/protocol-integration/lwm2m/#migration-of-the-lwm2m-devices" class="no-ajaxy">LWM2M > LWM2M connector device > Migration of the LWM2M devices</a> in the <i>Protocol integration guide</i>.</td>
+<td>DM-1866</td>
+<td>10.18.10.0</td>
+<td>Core</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>Two new LWM2M shell commands have been added.
+<br>- The new <code>executelegacy</code> command allows LWM2M execute requests with non-standard LWM2M parameters. The behavior of this operation resembles the semantics of the existing <code>execute</code> operation until version 10.15.
```
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/release-2024-0/devices-2024-0.md:61]: Still not true :-). "LWM2M execute requests" is a noun.

File: content/release-2024-0/platform-2024-0.md [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+<td>Core platform</td>
+<td>The platform performance has been improved by introducing a new cache for managed objects.</td>
+<td>MTM-50113</td>
+<td>10.18.9.0</td>
+<td>Core</td>
+</tr>
+
+<tr>
+<td>REST API</td>
+<td>At least one query parameter limiting the affected data is required to prevent accidental deletion of too many objects during a bulk delete operation.
```
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/release-2024-0/platform-2024-0.md:112]: It is in the important announcements of the previous releases.

[the remark this answers is not in the document]
Comment (rssag) [reply content/release-2024-0/platform-2024-0.md:112]: ah ok :)

File: content/release-2024-0/devices-2024-0.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (rssag) [reply content/release-2024-0/devices-2024-0.md:61]: right :D maybe "execution" instead of "execute" would facilitate parsing

**Pull request merged**